app/src/main/java/phuchh/sunasterisk/moviedb_41/data/model/Genre.kt

```diff
@@ -0,0 +1,13 @@
+package phuchh.sunasterisk.moviedb_41.data.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+class Genre{
+    @SerializedName("id")
+    @Expose
+    var mId: String? = null
```

Comment:
đối với kotlin, vì sử dụng property syntax lên không nên để prefix m ở các field
```suggestion
    var id: String? = null
```
tương tự với các class phía dưới